src/iframe/src/Iframe.js

```diff
@@ -314,21 +319,27 @@ class Iframe extends Component {
     let globals
 
     if (!providedGlobals) {
+      let canvasAPI = contextCanvasAPI
```

Comment:
We should use a ternary here `const canvasAPI = this.renderer === 'framebuffer' ? frameBufferCanvasAPI : contextCanvasAPI`

Reply:
Okeydokey.